### PR TITLE
Dark Souls Update

### PR DESCRIPTION
**Pull request by ghost**
-Added Dodge rolls
-Added Iframes
-Added parrying and reposting 
-Added boss souls
-Added smelting weapons from boss souls
-Added smelting new crafting materials
+Iron Ingots (Made from melting down tin cans with faggots)
+Leather  (Made from tanning old boots with a tanning knife made from iron ingots)
-Added TRUE RANDOM
-Removed Herobrine

### CODE REVIEW

File: ewcfg.py [hunk 1]
```diff
@@ -3225,8 +3253,36 @@ def wef_bow(ctn = None):
...
 		ctn.crit = True
-		ctn.slimes_damage = int(dmg * 10)
+		ctn.slimes_damage = int(dmg * 6)
+		
+# weapon effect function for "Dragon Claw"
 
+def wef_dclaw(ctn = None):
+	
+	user_mutations = ctn.user_data.get_mutations()
+	if mutation_id_fastmetabolsim in user_mutations or mutation_id_lightasafeather in user_mutations:
```
Comment (magicksRampage) [inline ewcfg.py]: typo fastmetabolism

File: ewcfg.py [hunk 2]
```diff
@@ -3225,8 +3253,36 @@ def wef_bow(ctn = None):
...
+                ctn.slimes_damage = int(ctn.slimes_damage * 1.2)
+                ctn.slimes_spent *= 0.5
+        else:
+                ctn.slimes_damage = int(ctn.slimes_damage * 1.5)
+                ctn.slimes_spent *= 1
+
+        #less slime cost and less damage = attacking faster I guess?
+	ctn.sap_damage = 5
+	ctn.sap_ignored = 10
+	if aim <= (random.randint(1,13):
```
Comment (magicksRampage) [inline ewcfg.py]: unmatched parenthesis. please at least run your code to make sure it works

Comment (magicksRampage) [reply ewcfg.py]: also aim is undefined

File: ewcfg.py [hunk 3]
```diff
@@ -3915,8 +3971,35 @@ def wef_bow(ctn = None):
...
+			"dragon claw",
+		],
+		str_crit = "{name_player} runs like a madman towards {name_target}, {name_target} swings but is deftly parried by {name_player}, {name_player} hoists their dragon claw into the air and ripostes {name_target} for massive damage ***!!!Critical Hit!!!***",
+		str_miss = "{name_player} swings but {name_target} is in the middle of a dodge roll and is protected by iframes. **!!Miss!!**",
+		str_equip = "You place the core of the dragon claw on your hand and it unfolds around it, conforming to the contour of your hands, claws protude out the end of your fingers as your hand completes its transformation into the *dragon claw*.",
+		str_name = "dragon claw",
+		str_weapon = "a dragon claw",
+		str_weaponmaster_self = "You are a rank {rank} master of the dragon claw.",
+		str_weaponmaster = "They are a rank {rank} master of the dragon claw.",
+		str_trauma_self = "Three smoldering claw marks are burned into your flesh, the flames wont seem to extinguish.",
```
Comment (magicksRampage) [inline ewcfg.py]: won't

File: ewcfg.py [hunk 4]
```diff
@@ -3915,8 +3971,35 @@ def wef_bow(ctn = None):
...
+		str_damage = random.choice(["{name_target} is slashed across the {hitzone}!!","{name_player} furiously slashes {name_target} across the {hitzone}!!","{name_player} flicks their fingers and a jet of flame ignites from the dragon claw, burning {name_target} in the {hitzone}!!"]),
+		str_duel = "**SLICE!! SWIPE!! SLASH!!** {name_player} and {name_target} cut the fuck out of eachother, a fire extinguisher is never more than a meter away.",
+		str_scalp = "The scalp is burning and doesn't look like it's gonna stop.",
+		fn_effect = wef_dclaw,
+		str_description = "It's the core of a Dragon Claw, it will morph around whatever hand it is held by granting them the power of the elusive GREEN EYES SLIME DRAGON. If you listen closely you can hear whines of the dragon soul as it remains perpetually trapped in the weapon.",
+		acquisition = acquisition_smelting,
+		stat = stat_dclaw_kills,
+                classes = [weapon_class_exploding],
+		sap_cost = 5,
+		captcha_length = 2)
```
Comment (magicksRampage) [inline ewcfg.py:4000]: this weapon absolutely needs a soft cooldown or a captcha

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewcfg.py (hunk 2 above)
Comment (magicksRampage) [reply ewcfg.py]: the "aim" variable is still undefined

Comment (ghost) [reply ewcfg.py]: May have accidentally changed that in the wrong file

**Pull request merged**